### PR TITLE
[WEB-970] Part 1: Add New Property sendMetaCapiEvents

### PR DESCRIPTION
**Pull request by scottkicks**
# 📲 What

Add new property to help determine if we need to send CAPI events to the backend
- sendMetaCapiEvents

# 🤔 Why

This new property will help tell us if we should send CAPI events

# 🛠 How

Added this property to the `Project` fragment.

```
...
sendMetaCapiEvents
...
```

# ✅ Acceptance criteria

- [x] Project page and Dashboard loads and no missing data. User can pledge/back/edit pledge/view comments/view updates, etc. No new functionality added.

### CODE REVIEW

**Comment by nativeksr** [comment]
### SwiftFormat found issues:

| File | Rules |
| ---- | ----- |
| KsApi/models/graphql/adapters/Project+FetchProjectQueryDataTests.swift:127:1 | warning: (indent) Indent code in accordance with the scope level. |


<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Comment by scottkicks** [comment]
@msadoon For some reason I'm getting this error locally in the auto generated GraphAPI file on build. Its keeping me from testing the currently failing file. I'm wondering if you see the same on your end.
Screen Shot 2023-02-22 at 12 24 19 PM

**Comment by msadoon** [comment]
> 

Weird, I pulled down your branch and it just built right away. Not sure...

**Comment by sclampet** [comment]
> > 
> 
> 
> 
> Weird, I pulled down your branch and it just built right away. Not sure...

Did KsApi test run?

**Comment by msadoon** [comment]
> > > 
> > 
> > 
> > Weird, I pulled down your branch and it just built right away. Not sure...
> 
> Did KsApi test run?

Yea so it took me bit to find the error, related to some test templates we were using, the last commit should have the fix. :)

File: Kickstarter.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -2077,6 +2082,11 @@
...
+		606F214B2994158500BA5CDF /* TriggerCapiEventInput.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = TriggerCapiEventInput.swift; sourceTree = "<group>"; };
+		606F215A299D3EF800BA5CDF /* GraphAPI.TriggerCapiEventInput+TriggerCapiEventInputTests.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = "GraphAPI.TriggerCapiEventInput+TriggerCapiEventInputTests.swift"; sourceTree = "<group>"; };
+		606F215D299D414800BA5CDF /* GraphAPI.TriggerCapiEventInput+TriggerCapiEventInput.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = "GraphAPI.TriggerCapiEventInput+TriggerCapiEventInput.swift"; sourceTree = "<group>"; };
+		606F2165299D456D00BA5CDF /* TriggerCapiEvent.graphql */ = {isa = PBXFileReference; lastKnownFileType = text; path = TriggerCapiEvent.graphql; sourceTree = "<group>"; };
+		606F2167299D45F900BA5CDF /* TriggerCapiEventInput.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = TriggerCapiEventInput.swift; sourceTree = "<group>"; };
```
Comment (msadoon) [inline Kickstarter.xcodeproj/project.pbxproj:2089]: Weird that it's showing multiple times...I checked the finder folder and xcode and the file is there only once.

**Review by msadoon — APPROVED** [review]
Great work!

File: Kickstarter.xcodeproj/project.pbxproj (hunk 1 above)
Comment (scottkicks) [reply Kickstarter.xcodeproj/project.pbxproj:2089]: ok i will keep this in mind as I work through Part 2 and see if git is caching it for some reason.

**Pull request merged**